src/main/java/com/clanki/commands/ReviewCommand.java

```diff
@@ -0,0 +1,49 @@
+package com.clanki.commands;
+
+import com.clanki.objects.Flashcard;
+import com.clanki.objects.FlashcardList;
+import com.clanki.ui.Ui;
+
+import java.util.ArrayList;
+import java.util.Objects;
+
+public class ReviewCommand extends Command {
+    private static void reviewCurrentFlashcard(Ui display, Flashcard flashcard) {
+        boolean isCurrentCardPassed = false;
+        display.print("Let's see the next flashcard!");
+        while (!isCurrentCardPassed) {
+            display.print("Question for the current flashcard is: ", flashcard.getQuestion());
+            display.print("Are you ready to view the answer?");
+            String viewAnswerCommand = display.getUserCommand();
+            if (Objects.equals(viewAnswerCommand, "answer") || Objects.equals(viewAnswerCommand, "a")) {
+                display.print(flashcard.getAnswer(), "Have you got this correct?");
+                String ifCorrectAnswerCommand = display.getUserCommand();
```

Comment:
I don't think this is a very good name for this variable. I think `answerIfCorrect` is a better name.